std/utf.d

```diff
@@ -4394,9 +4441,26 @@ if (isSomeChar!C)
                     this.buf = buf;
                 }
 
+                static if (isBidirectionalRange!R)
+                {
+                    this(return R r, ushort frontPos, ushort frontFill,
+                         ushort backPos, ushort backFill, C[4 / C.sizeof] buf)
```

Comment:
Again, I don't see any calls to this constructor. Is this needed?

Reply:
@RazvanN7  ,sorry for the delay and thanks for pointing it out. Right now the constructor is not used, but I discovered a problem in the meantime.
Consider this piece of code
```
wchar[] s = ['Ă', 'ț'];
auto r = s.byUTF!char;
r.popBack;
auto rCopy = r.save
//right now r == [0xc4, 0x82, 0xc8];
//and rCopy == [0xc4, 0x82];
```
It is fixable by having different save functions for forward and bidirectional ranges.
(And this way I will need the new constructor).
Is it ok if I do this, right?
I also pushed a commit to see what I mean, now I have to fix the rebasing.

Reply:
I see, so the saved back buff is not considered in the initial version of `save`.